src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -94,9 +75,15 @@ public ProjectsWrapper queryCristinProjectsIntoWrapperObjectWithAdditionalMetada
         Map<String, String> requestQueryParams) throws ApiGatewayException {
 
         long startRequestTime = System.currentTimeMillis();
-        List<NvaProject> nvaProjects = importProjectsFromCristin(requestQueryParams);
+        HttpResponse<InputStream> response = queryProjects(requestQueryParams);
```

Comment:
Require explicitly that you have a UTF-8 encoded content by altering fetchGetResult to
```
   @JacocoGenerated
    protected HttpResponse<String> fetchGetResult(URI uri) {
        HttpRequest httpRequest = HttpRequest.newBuilder(uri).build();

        return attempt(() -> client.send(httpRequest, BodyHandlers.ofString(StandardCharsets.UTF_8))).orElseThrow();
    }
```
Either way or the other we require it implicitly when we use IoUtils.streamToString()

Reply:
You want me to change the response type body to String instead of InputStream? Do you want me to do it for both query/search and lookup? Should i do that in separate PR or in this one? Personally I think it should be its own PR with an optional issue

Reply:
OK

Reply:
I have created issue for it